src/smc-webapp/frame-editors/latex-editor/actions.ts

```diff
@@ -674,6 +681,30 @@ export class Actions extends BaseActions<LatexEditorState> {
     }
   }
 
+  make_timestamp(time: number, force: boolean): number | undefined {
+    return force ? undefined : time || this._last_save_time;
+  }
+
+  async word_count(time: number, force: boolean): Promise<void> {
+    // only run word count if at least one panel is active
```

Comment:
active unfortunately means "focused", not "exists".   Your code will work fine, bu the comment should be changed from "active" to "exists".  

It would be better to define a new method `_has_frame_id_of_type` with similar code to `_get_most_recent_active_frame_id_of_type` (even the same with == null for now would be fine) in the base class and call that instead, to make the intention clearer.

Reply:
ok, I can add that function